.travis.yml

```diff
@@ -1,4 +1,5 @@
 language: node_js
 node_js:
     - stable
-
+script:
+  - node ./validation/markets-validator.js
```

Comment:
By default, [travis runs `npm test`](https://docs.travis-ci.com/user/languages/javascript-with-nodejs#Default-Test-Script) so simply make sure that `npm test` also executes the validation instead.